ChaosMod/Effects/db/Player/PlayerLeanedCamera.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+	Effect by Moxi
+*/
+
+#include <stdafx.h>
+
+static Camera leanedCamera = 0;
+static float angle = 0;
+
+static void UpdateCamera()
+{
+    Vector3 coord = GET_GAMEPLAY_CAM_COORD();
+    Vector3 rot = GET_GAMEPLAY_CAM_ROT(2);
+    float fov = GET_GAMEPLAY_CAM_FOV();
+    SET_CAM_PARAMS(leanedCamera, coord.x, coord.y, coord.z, rot.x, angle, rot.z, fov, 0, 1, 1, 2);
+}
+
+static void OnStart()
+{
+    angle = g_Random.GetRandomInt(0, 1) ? g_Random.GetRandomFloat(45.f, 135.f) : g_Random.GetRandomFloat(-135.f, -45.f);
```

Comment:
angle = g_Random.GetRandomFloat(45.f, 135.f) * (g_Random.GetRandomInt(0, 1) == 0 ? 1 : -1);

Reply:
but that just may be my ocd :D